js/web/lib/backend-wasm.ts

```diff
@@ -22,9 +22,9 @@ export const initializeFlags = (): void => {
   }
   env.wasm.numThreads = Math.min(4, env.wasm.numThreads);
 
-  if (typeof env.wasm.loggingLevel !== 'string' ||
-      ['verbose', 'info', 'warning', 'error', 'fatal'].indexOf(env.wasm.loggingLevel) === -1) {
-    env.wasm.loggingLevel = 'warning';
+  if (typeof env.loggingLevel !== 'string' ||
```

Comment:
since loggingLevel is now for both webgl and wasm, we need to move out this initialization out of webassembly. I'm wondering if a user enables only webgl, loggingLevel is correctly set for webgl?

Reply:
webgl still uses the old onnxjs logger which has 'info' set as default logging level if logginglevel flag is unset, shall we change it to warning to match the default level for ort? then we can move this initialization to a common path.

Reply:
since this flag is in `env`, it is expected to be used in onnxruntime-node and onnxruntime-react-native. So setting it in onnxruntime-web is not a good idea. I think there are 2 ways for initialization of `env.loggingLevel`:
1. initialize this variable in onnxruntime-common.
2. does not initialize; when using it, always wrap with a check and a fallback if it's not set.

Also, we have 2 different way to deal with the variable setting. when user code call `ort.env.loggingLevel = 'a_bad_value'`, we can:
1. throw exception
2. silently ignore this assignment, keep `ort.env.loggingLevel`'s previous value
3. allow assign `ort.env.loggingLevel` to `a_bad_value`, and deal with it in each backend implementation (then it's depend on each backend's implementation for the behavior)

Reply:
Yes i agree that moving it to common is the best way to go if we assume the same default level for both ort and ort web. Since eventually we want all backends' severity levels to match with ort, i'd prefer checking and setting to happen in common as well, each backend can have different format of logging but it doesnt need to care about how this logging level is set to, all backends should assume whatever they receive is a correct value. If we all agree, i can make the changes.

Reply:
renamed loggingLevel to logLevel and moved its validation to env-impl.